Document/0x05h-Testing-Platform-Interaction.md

```diff
@@ -141,14 +141,28 @@ Check permissions to make sure that the app really needs them and remove unneces
 
 Go through the permissions with the developer to identify the purpose of every permission set and remove unnecessary permissions.
 
-Besides going through the AndroidManifest.xml file manually, you can also use the Android Asset Packaging tool to examine permissions.
+Besides going through the AndroidManifest.xml file manually, you can also use the Android Asset Packaging tool to examine the permissions of an APK file.
+
+> aapt comes with the Android SDK within the build-tools folder. It requires an APK file as input. You may list the APKs in the device by doing `adb shell pm list packages -f | grep -i <keyword>` as seen in "[Listing Installed Apps](0x05b-Basic-Security_Testing.md#listing-installed-apps "Listing Installed Apps")".
```

Comment:
```suggestion
> aapt comes with the Android SDK within the build-tools folder. It requires an APK file as input. You may list the APKs in the device by running `adb shell pm list packages -f | grep -i <keyword>` as seen in "[Listing Installed Apps](0x05b-Basic-Security_Testing.md#listing-installed-apps "Listing Installed Apps")".
```

Reply:
Maybe the quotes around the URL are not needed?

Reply:
This is how we are referencing to chapters and sections now. -> (Style Guide) 4. References Within The Guide ^^

So I will just accept your suggestion as-is :)